noir-projects/aztec-nr/aztec/src/macros/dispatch/mod.nr

```diff
@@ -158,7 +158,7 @@ comptime fn str_size_in_fields(typ: Type) -> Option<u32> {
 }
 
 comptime fn struct_size_in_fields(typ: Type) -> Option<u32> {
-    typ.as_struct().map(|typ: (StructDefinition, [Type])| {
```

Comment:
Find and replace is colouring outside the lines here.

Reply:
wdym?

Reply:
Ah sorry, I did this on mobile so looks like I linked to the wrong file. You're also modifying the noir stdlib in this PR.

Reply:
I've fixed this by merging master